pkg/tsdb/azuremonitor/loganalytics/azure-log-analytics-datasource_test.go

```diff
@@ -100,6 +101,7 @@ func TestBuildLogAnalyticsQuery(t *testing.T) {
 	tests := []struct {
 		name                   string
 		fromAlert              bool
+		setBasicLogsEnabled    bool
```

Comment:
I think you need to set this value on the `dsInfo` in order to appropriately use it.